tests/test_log_likelihood.py

```diff
@@ -0,0 +1,88 @@
+"""Assert values log-likelihood function under ambiguity (speed-up)"""
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+import respy as rp
+from respy.likelihood import get_crit_func
+from respy.simulate import get_simulate_func
+
+subdir = Path(f"{os.environ['PROJECT_ROOT']}/tests/resources")
+
+
+# Test: Assert likelihood value for different model specifications (options, params)
```

Comment:
Why is this comment not a regular docstring of the function?

Reply:
Habit from a course I've taken :) 
Of course, going forward I will docstring test functions appropriately.